.github/workflows/ci_test-slow.yml

```diff
@@ -57,15 +57,15 @@ jobs:
 
     - name: Tests
       run: |
-        coverage run --source pytorch_lightning -m pytest tests -v --junitxml=junit/test-results-${{ runner.os }}-${{ matrix.python-version }}.xml
+        coverage run --source pytorch_lightning -m pytest tests -v --junitxml=junit/test-results-${{ runner.os }}-py${{ matrix.python-version }}.xml
       env:
         PL_RUN_SLOW_TESTS: 1
 
     - name: Upload pytest test results
       uses: actions/upload-artifact@v2
       with:
-        name: pytest-results-${{ runner.os }}-${{ matrix.python-version }}
-        path: junit/test-results-${{ runner.os }}-${{ matrix.python-version }}.xml
+        name: pytest-results-${{ runner.os }}-py${{ matrix.python-version }}
+        path: junit/test-results-${{ runner.os }}-py${{ matrix.python-version }}.xml
```

Comment:
does this need?:

```suggestion
        path: junit/test-results-${{ runner.os }}-py${{ matrix.python-version }}.xml
        if-no-files-found: error
```

Reply:
it shall prevent silent missing reports...

Reply:
I'm asking if we should add it here too

Reply:
I would say yes

Reply:
Committing my suggestion then